README.md

```diff
@@ -558,6 +558,8 @@ const uiSchema = {
 };
 ```
 
+In addition it is valid to mark an array as "required".  This means the user must select at least one item to add to the array.
```

Comment:
Good, @crumblix would you mind updating the patch accordingly if you agree with this decision? :)

Thanks a bunch :+1: